website/app/components/doc/table-of-contents/collapsible-item.hbs

```diff
@@ -0,0 +1,12 @@
+<div class="doc-table-of-contents__folder">
+  <button
+    type="button"
+    aria-controls={{this.contentId}}
+    aria-expanded={{this.isOpen}}
+    class="doc-table-of-contents__button {{if this.isOpen 'open'}}"
+    {{on "click" this.toggleIsOpen}}
+  >{{@name}}</button>
+  {{#if this.isOpen}}
+    <Doc::TableOfContents @structuredPageTree={{@item.children}} @depth={{add @depth 1}} id={{this.contentId}} />
```

Comment:
issue: we increment the depth when calling this component, doing it again here will result in a double increment (and double spacing).
```suggestion
    <Doc::TableOfContents @structuredPageTree={{@item.children}} @depth={{@depth}} id={{this.contentId}} />
```